src/components/ui/sheet.tsx

```
@@ -29,7 +31,7 @@ const SheetOverlay = ({
   return (
     <SheetPrimitive.Overlay
       className={cn(
-        'fixed inset-0 z-50 bg-black/50 data-[state=closed]:animate-out data-[state=closed]:fade-out-0 data-[state=open]:animate-in data-[state=open]:fade-in-0',
+        'fixed inset-0 z-40 bg-black/50 data-[state=closed]:animate-out data-[state=closed]:fade-out-0 data-[state=open]:animate-in data-[state=open]:fade-in-0',
```

ESLint

[error] Unknown class detected: data-[state=closed]:animate-out

(siberiacancode-tailwind/no-unknown-classes)